### PR TITLE
EN 1276 fix add new role

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
retest this please

**Comment by entando-service** [comment]
ok to test

**Comment by entando-service** [comment]
retest this please

File: src/ui/roles/detail/DetailRoleTable.js [hunk 1]
```diff
@@ -55,14 +54,16 @@ DetailRoleTable.propTypes = {
   role: PropTypes.shape({
     code: PropTypes.string,
     name: PropTypes.string,
-    permissions: PropTypes.arrayOf(PropTypes.string),
+    permissions: PropTypes.shape({}),
```
Comment (alessioscalici) [inline src/ui/roles/detail/DetailRoleTable.js]: This is unneeded

**Comment by nicpuddu** [comment]
retest this please

**Comment by nicpuddu** [comment]
test this please

**Pull request merged**